services/configuration/src/schema.rs

```diff
@@ -52,15 +52,70 @@ lazy_static! {
         &Hash::zero(),
         &Signature::zero(),
     ).into_bytes();
+    static ref VOTE_DECISION_BYTES_MARKER: Vec<u8> = b"consent".to_vec();
+    static ref VOTE_AGAINST_DECISION_BYTES_MARKER: Vec<u8> = b"dissent".to_vec();
 }
 
-/// A functional equivalent to `Option<Vote>` used to store votes in the service schema.
+/// A enum used to represent different kinds of vote, `Vote` and `VoteAgainst` transactions.
+#[derive(Clone, Debug, PartialEq, Serialize, Deserialize)]
+pub enum VotingDecision {
```

Comment:
To make the change backward-compatible, it would be necessary to change [the enum representation by `serde`](https://serde.rs/enum-representations.html). Although untagged technically can work, I suggest using internal tagging, i.e., `#[serde(tag = "type")]`. Also, I'm not quite sure about variant names: should they be renamed, and if so, how. A possible suggestion is `"yea" | "nay"` (used in parliaments in English-speaking countries; see [Wikipedia](https://en.wikipedia.org/wiki/Division_of_the_assembly)).

Reply:
if we change variant names to "yea" and "nay", maybe we can use "vote_for" for tag name?
Or use "consent" and "dissent"?

Reply:
Rename to Yea and Nay, use "vote_for" for internal tag.

Done